src/annotator/components/ModalDialog.tsx

```diff
@@ -0,0 +1,87 @@
+import classnames from 'classnames';
+import type { ComponentChildren } from 'preact';
+import { useEffect, useMemo, useRef } from 'preact/hooks';
+
+type DialogProps = {
+  closed: boolean;
+  children: ComponentChildren;
+  onClose: () => void;
+  'data-testid'?: string;
+};
+
+const NativeDialog = ({
```

Comment:
This should be `function NativeDialog(...)` following the convention to use `function` for top-level functions in a module. Same with `FallbackDialog` below.

Reply:
Oops! You are absolutely right 😅